scipy/stats/__init__.py

```diff
@@ -7,8 +7,30 @@
 
 .. currentmodule:: scipy.stats
 
-This module contains a large number of probability distributions as
-well as a growing library of statistical functions.
+This module contains a large number of probability distributions,
+summary and frequency statistics, correlation functions and statistical
+tests, masked statistics, kernel density estimation, quasi-Monte Carlo
+functionality, and more.
+
+Statistics is a very large area, and there are topics that are out of scope
+for SciPy and are covered by other packages. Some of the most important ones
+are:
+
+- `statsmodels <https://www.statsmodels.org/stable/index.html>`__:
+  regression, linear models, time series analysis, extensions to topics
+  also covered by ``scipy.stats``.
+- `Pandas <https://pandas.pydata.org/>`__: tabular data, time series
+  functionality, interfaces to other statistical languages.
+- `PyMC3 <https://github.com/pymc-devs/pymc3>`__: Bayesian statistical
```

Comment:
Is the repo preferable to https://docs.pymc.io/?

Reply:
I didn't find the other site so quickly, will change.